src/views/RecIM/components/Header/index.jsx

```diff
@@ -56,10 +56,10 @@ const Header = ({ match, team, activity, admin, children }) => {
           {match && (
             <RecIMBreadcrumb>
               {largeWidth
-                ? `Match: ${match?.Team[0]?.Name ?? 'TBD'} vs 
-              ${match?.Team[1]?.Name ?? 'TBD'}`
-                : `Match: ${truncate(match?.Team[0]?.Name ?? 'TBD')} vs ${truncate(
-                    match?.Team[1]?.Name ?? 'TBD',
+                ? `Match: ${match.Team?.[0]?.Name ?? 'TBD'} vs 
```

Comment:
Does doing Match.Team.[0] work as an index?

Reply:
not my implementation